Aquality.WinAppDriver/src/Aquality.WinAppDriver/Elements/Actions/IMouseActions.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Aquality.WinAppDriver.Elements.Interfaces;
+using OpenQA.Selenium.Interactions;
+
+namespace Aquality.WinAppDriver.Elements.Actions
+{
+    /// <summary>
+    /// Provides methods representing basic mouse actions against the current element.
+    /// Current element's coordinates are user as last known mouse coordinates.
+    /// </summary>
+    public interface IMouseActions : WinAppDriver.Actions.IMouseActions
+    {
+        /// <summary>
+        /// Performs a drag-and-drop operation on the current element; drops it on target element.
+        /// </summary>
+        /// <param name="target">The element on which the drop is performed.</param>
+        void DragAndDrop(IElement target);
+
+        /// <summary>
+        /// Performs a drag-and-drop operation on the current element to a specified offset.
+        /// </summary>
+        /// <param name="offsetX">The horizontal offset to which to move the mouse.</param>
+        /// <param name="offsetY">The vertical offset to which to move the mouse.</param>
+        void DragAndDropToOffset(int offsetX, int offsetY);
```

Comment:
I am not sure that we have to add 'ToOffset' at the end of the method. List of parameters shows what will be done by this method. What do you think?

Reply:
discussed. this way is better in sense of readability